apps/entry/models.py

```diff
@@ -917,6 +914,7 @@ def get_figure_excel_sheets_data(cls, figures: models.QuerySet):
             created_by__full_name='Created by',
             last_modified_by__full_name='Updated by',
             event_codes='Event codes (Code:Type)',
+            locations='Locations (Location:Location Name:Location Accuracy:Type of Point)',
```

Comment:
Maybe we do use something like this instead?
```suggestion
            locations='Locations (Location:Lat,Lon:LocationAccuracy:TypeOfPoint)',
```